docs/how/search.md

```diff
@@ -1,17 +1,76 @@
-# Search Guide
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
 
-## Introduction 
+# About DataHub Search
 
-The search bar is one of the means of finding data in Datahub. In this document, we discuss more effective ways of finding information beyond doing a standard keyword search. This is because keyword searches can return results from almost any part of an entity.
+<!-- All Feature Guides should begin with `About DataHub ` to improve SEO -->
 
-### Search in Specific Fields
+<!-- 
+Update feature availability; by default, feature availabilty is Self-Hosted and Managed DataHub
 
-The following examples are in the format of  
-X: *typical question* :  
+Add in `saasOnly` for Managed DataHub-only features
+ -->
+
+<FeatureAvailability/>
+
+The **search bar** is an important mechnaism for discovering data assets in DataHub. From the search bar, you can find Datasets, Columns, Dashboards, Charts, Data Pipelines, and more. Simply type in a term and press 'enter'. 
+
+![](https://github.com/datahub-project/static-assets/blob/main/imgs/search-landingpage.png?raw=true)
```

Comment:
Small nit-pick for consistency with other guides & improved readability - please use HTML to embed images so we can size down (70%) & center-align: 

```
<p align="center">
  <img width="70%"  src="https://github.com/datahub-project/static-assets/blob/main/imgs/search-landingpage.png?raw=true"/>
</p>
```